conditional/__init__.py

```diff
@@ -190,8 +189,7 @@ def route_errors(error, user_dict=None):
     return render_template('errors.html',
                            error=error_desc,
                            error_code=code,
-                           event_id=g.sentry_event_id,
-                           public_dsn=sentry.client.get_public_dsn('https'),
```

Comment:
This line being removed makes me think all JS will break as the JS raven client won't know where to send errors.

Reply:
```git:rm-raven  1M 
(venv) !!! 34 16:19 bionicpi@achlys:conditional $ grep -r Raven conditional/
git:rm-raven  1M 
(venv) !!! 35 16:20 bionicpi@achlys:conditional $ grep -r raven conditional/
git:rm-raven  1M 
(venv) !!! 36 16:20 bionicpi@achlys:conditional $ 
```
... how is raven even set up in the js?

Reply:
Check the `frontend` folder
```
frontend/javascript/modules/errorReport.js:import Raven from 'raven-js';
frontend/javascript/app.js:import Raven from 'raven-js';
```
```
frontend/javascript/modules/errorReport.js:import Raven from 'raven-js';
frontend/javascript/modules/errorReport.js:    this.btn.addEventListener('click', () => this._invokeRavenModal());
frontend/javascript/modules/errorReport.js:  _invokeRavenModal() {
frontend/javascript/modules/errorReport.js:    Raven.showReportDialog({
frontend/javascript/app.js:// Install Raven to send errors to Sentry
frontend/javascript/app.js:import Raven from 'raven-js';
frontend/javascript/app.js:Raven
frontend/javascript/app.js:  Raven.captureException(err.reason);
```